templates/config.edn

```diff
@@ -1,137 +1,160 @@
 {:meta/version 1
 
- ;; Currently, we support either "Markdown" or "Org".
- ;; This can overwrite your global preference so that
- ;; maybe your personal preferred format is Org but you'd
- ;; need to use Markdown for some projects.
- ;; :preferred-format ""
-
- ;; Preferred workflow style.
- ;; Value is either ":now" for NOW/LATER style,
- ;; or ":todo" for TODO/DOING style.
+ ;; Set the preferred format.
+ ;; Available Options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ ;; :preferred-format "Markdown"
+
+ ;; Set the preferred workflow style.
+ ;; Available options:
+ ;; - ":now" for NOW/LATER style (default)
+ ;; - ":todo" for TODO/DOING style
  :preferred-workflow :now
 
- ;; The app will ignore those directories or files.
- ;; E.g. :hidden ["/archived" "/test.md" "../assets/archived"]
+ ;; Exclude directories/files.
+ ;; Example usage: 
+ ;; :hidden ["/archived" "/test.md" "../assets/archived"]
  :hidden []
 
- ;; When creating the new journal page, the app will use your template if there is one.
- ;; You only need to input your template name here.
- :default-templates
- {:journals ""}
-
- ;; Set a custom date format for journal page title
- ;; Example:
- ;; :journal/page-title-format "EEE, do MMM yyyy"
+ ;; Define the default journal page template.
+ ;; Enter the template name between the quotes.
+ :default-templates {:journals ""}
+
+ ;; Set a custom date format for the journal page title.
+ ;; Default value: "MMM do, yyyy"
+ :journal/page-title-format "MMM do, yyyy"
+
+ ;; Specify the journal filename format.
+ ;; !Warning:
+ ;;   - Changing this setting may cause issues if not configured correctly.
+ ;;     This configuration option is not retroactive and
+ ;;     will only affect newly created journals.
+ ;; You may use any valid date format string.
+ ;; Default value: "yyyy-MM-dd"
+ ;; :journal/file-name-format "yyyy_MM_dd"
```

Comment:
why is the default value different from the value here?

Reply:
And I think it's better to uncomment this line for default value

Reply:
I still think this should be changed to have an uncommented version of the default value being set

Reply:
Update: After discussing on Discord, and realizing this format is commonly used for compatibility with things like Obsidian, I think that leaving this here is fine, but it'd be nice to add a comment like:

```
 ;; Default value: "yyyy-MM-dd"
 ;; Example usage:
 ;; :journal/file-name-format "yyyy_MM_dd" ;This format is compatible with Obsidian and others
 ```

Reply:
But this is defo nonblocking and it sounds like this isn't really worth expounding on